clients/linux/src/filetree.rs

```diff
@@ -419,12 +425,16 @@ impl FileTreePopup {
         let tsel = t.get_selection();
         let tmodel = t.get_model().unwrap();
 
+        let is_root = tsel.iter_is_selected(&match tmodel.get_iter_first() {
```

Comment:
you should put this in an `if let` rather than returning early. You can put the rest of this fn in the if let, because that's what's actually happening here